.travis.yml

```diff
@@ -3,10 +3,22 @@ notifications:
 
 language: python
 
-env:
-  - TOX_ENV=beets_master COVERAGE=1
-  - TOX_ENV=beets_1.4.1
-  - TOX_ENV=flake8
+branches:
+  only:
+    master
```

Comment:
I don’t quite understand the rationale behind this? Also seems like it has no effect since there is a build of your PR.

Reply:
I don't understand it anymore either, and have missed it when reviewing. I added this when I set up travis on my repo, but the original commit message (which I squashed when rebasing) isn't very helpful. Anyway, I'll remove those lines.